src/lib/common/string.h

```diff
@@ -31,6 +31,7 @@
 #include <vector>
 
 #include "common/limits.h"
+#include "common/globals.h"  // FIXME P10: PARANOID_JSON_INDENT
```

Comment:
FIXME P10 ... ?
If it's a P10, it should be fixed before we merge the PR

Reply:
Not in this PR, but in a soon next one. See https://github.com/telefonicaid/fiware-orion/issues/2760#issuecomment-337334053.

Reply:
Ok

NTC